app/src/main/java/com/aacademy/homework/data/api/ApiSource.kt

```diff
@@ -0,0 +1,14 @@
+package com.aacademy.homework.data.api
+
+import com.aacademy.homework.data.model.Actor
+import com.aacademy.homework.data.model.Genre
+import com.aacademy.homework.data.model.JsonMovie
+
+interface ApiSource {
```

Comment:
remove

Reply:
can't remove it. because ApiSourceImpl gets MoviesService in the constructor and can't implement MoviesService instead of ApiSource.